src/SFA.DAS.Tools.Servicebus.Support.Web/Controllers/MessageListController.cs

```diff
@@ -72,27 +78,52 @@ public async Task<IActionResult> ReceiveMessages(string queue)
             return RedirectToAction("Index");
         }
 
-        public async Task<IActionResult> AbortMessages()
+        [HttpPost]
+        public async Task<IActionResult> AbortMessages([FromBody] SelectedMessages selectedMessages)
         {
-            var response = await _getMessagesQuery.Handle(new GetMessagesQuery()
+            var response = await _getMessagesByIdQuery.Handle(new GetMessagesByIdQuery()
                 {
                     UserId = _userService.GetUserId(),
-                    SearchProperties = new SearchProperties()
-
+                    Ids = selectedMessages.Ids
                 });
 
-            foreach (var msg in response.Messages)
+            await _messageService.AbortMessages(response.Messages, selectedMessages.Queue);            
+
+            return RedirectToAction("Index", "Home");
+        }
+
+        public async Task<IActionResult> EndSession(string queue)
```

Comment:
I think we need some better terminology for this action so it's more easily understood by the user what it will do